source/reference/api/progress.txt

```diff
@@ -28,11 +28,25 @@ Request
 Response
 --------
 
-All response fields are wrapped in a top-level ``progress`` object. The
-endpoint returns either an updated status or an error.
+The ``progress`` endpoint returns etiher an updated status or an error.
+
+Successful Response
+~~~~~~~~~~~~~~~~~~~
+
+If ``mongosync`` successfully gets the status of the sync process, 
+all response fields are wrapped in a top-level ``progress`` object 
+with the following fields:
 
 .. include:: /includes/api/tables/progress-response.rst
 
+Error Response
+~~~~~~~~~~~~~~
+
+If ``mongosync`` encounters an error, the ``progress`` endpoint returns 
+an ``error`` object with the following field:
+
+.. include:: /includes/api/tables/progress-error-response.rst
```

Comment:
This is not quite right. In the case of an error, the response is an object with only an error key, like this:

```
{
  "error": "Some description of the error"
}
```

There's no `errorDescription` field at all, and the object isn't nested. Sorry if this was unclear in the ticket!

Reply:
Fixed